### PR TITLE
Fix shitcoins

### PR DESCRIPTION
**Pull request by terrysahaidak**
Fixes RNBW-3974
Figma link (if any):

## What changed (plus any additional context for devs)

We are fetching prices from 3 places:
1. Zerion
2. Covalent
3. Uniswap

The code right now prefers prices from zerion over prices from covalent.

But turns out we were also subscribing to prices from uniswap and treating them same as prices from zerion which is wrong.

We no longer need prices from uniswap, this is why I removed uniswap prices entirely.

Another change in this PR is that we store in redux hidden tokens before the actual assets so now it will only render actual visible rows. It's hard to reproduce, but it's happened to be a few times.

## Screen recordings / screenshots

|Before|After|
|--|--|
| vitalik's wallet with shitcoins | vitalik's wallet without shitcoins |

Also here is a recording from Samsung a12 after clearing all the caches:

https://user-images.githubusercontent.com/7809008/180520336-09ce1e01-7765-4b6b-a289-a943b7ac205c.mp4

## What to test

- Shitcoins should not have prices
- There should be prices in general for popular assets
- It should not longer flash the user shitcoins (at least with prices)
- Should be less blanks while scrolling (since we no longer receive prices for shitcoins in background)
- All the critical paths should work, so it needs a full regression

## Final checklist

- [x] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

File: src/redux/data.ts [hunk 1]
```diff
@@ -1203,7 +1202,7 @@ export const addressAssetsReceived = (
  *
  * @param addresses The asset addresses to subscribe to.
  */
-const subscribeToMissingPrices = (addresses: string[]) => (
+export const subscribeToMissingPrices = (addresses: string[]) => (
```
Comment (terrysahaidak) [inline src/redux/data.ts:1205]: just to fix the linter complain of not used function but this is just temporary

**Review by jinchung — APPROVED** [review]
🌮

**Comment by jinchung** [comment]
Will be part of: https://github.com/rainbow-me/rainbow/pull/3797

**Pull request closed without merging**